openshift/topics/advanced_concepts.adoc

```diff
@@ -1,11 +1,16 @@
-== Advanced Concepts
+== Performing advanced procedures
 
-These cover additional configuration topics, such as seting up keystores and a truststore for the {project_name} server, creating an administrator account, an overview of available {project_name} client registration methods, and guidance on configuring clustering.
+[role="_abstract"]
+This chapter describes advanced procedures, such as setting up keystores and a truststore for the {project_name} server, creating an administrator account, as well as an overview of available {project_name} client registration methods, and guidance on configuring clustering.
 
-=== Requirements and Deploying xref:../introduction/introduction.adoc#passthrough-templates[Passthrough TLS Termination] {project_name} Templates
+=== Deploying passthrough TLS termination templates
+
+You can deploy using these templates, which use HTTPS and JGroups keystores and a truststore for the {project_name} server.
```

Comment:
@andymunro Maybe replace `You can deploy using these templates, which use HTTPS and JGroups keystores and a truststore for the {project_name} server.` with `You can deploy using these templates. They require HTTPS, JGroups keystores and the {project_name} server truststore to already exist, and therefore can be used to instantiate the {project_name} server pod using your custom HTTPS, JGroups keystores and {project_name} server truststore.` ?

The difference compared to your original version is it doesn't say `which use HTTPS, JGroups... etc`. All of the RH-SSO templates, we ship use / require the HTTPS, JGroups keystore and the RH-SSO truststore.

The only difference is:
* For the re-encrypt (x509) ones -- all these keystores & truststore are autogenerated using the OpenShift's x509 service,
* While for the passthrough ones, the user needs to create them manually and provide the location (all the details, like credentials etc) via the environment variables, passed at the moment of template instantiation.

Makes sense?

Reply:
Okay sounds good.